<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionSelect</class>
 <widget class="QDialog" name="PartitionSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Partition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pc-bootconfig.qrc">
    <normaloff>:/icons/rename.png</normaloff>:/icons/rename.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Select a disk/partition for %1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_hd"/>
   </item>
   <item>
    <widget class="QListWidget" name="list_part"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="pc-bootconfig.qrc">
         <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_ok">
       <property name="text">
        <string>Done</string>
       </property>
       <property name="icon">
        <iconset resource="pc-bootconfig.qrc">
         <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pc-bootconfig.qrc"/>
 </resources>
 <connections/>
</ui>
